WebRoot/js/zimbraMail/briefcase/controller/ZmBriefcaseController.js

```diff
@@ -1416,7 +1416,26 @@ ZmBriefcaseController.prototype.handleModifyNotify =
 function(modifies){
     var view = this._listView[this._currentViewId];
     if (view) {
-        view.deselectAll();
+        var removeSelection = false;
+        var selectedItems = view.getSelection();
+        var selecteItemsId = [];
+
+        for (var i = 0; i < selectedItems.length; i++) {
+            selecteItemsId.push(selectedItems[i].id);
+        }
+
+        for (var i = 0; i < modifies.doc.length; i++) {
```

Comment:
Can we skip the `for` block if `selecteItemsId` is an empty array?
In addition, can we skip the process from line 1421 to 1438 if `selectedItems.length` is 0 ?

Reply:
Fixed.